gui/hud.gui

```diff
@@ -616,7 +616,7 @@ widget = {
 			}
 
 			icon = {
-				size = { 100% 100% }
+				size = { 100% 110% }
```

Comment:
It changes the size of the icon but doesn't affect the entire tab.
For example, it doesn't move the Royal Court button down.
You have to change the size of `main_tabs` widget.

Reply:
You are right. I didn't test it with Royal Court on. 
With the background kept at 100% and changing only the main_tabs widget the result is much more acceptable.
![image](https://user-images.githubusercontent.com/35965798/155419854-ac08c70d-66ce-4b4a-8f8a-e2ca488ed918.png)

Thank you!